CODE_OF_CONDUCT.md

```
@@ -0,0 +1,57 @@
+
+1. **Correction** — A private or public request to change behavior  
+2. **Warning** — A formal warning regarding unacceptable behavior  
+3. **Temporary Restriction** — Temporary suspension from participation  
+4. **Permanent Ban** — Permanent removal from the community  
+
+The severity and duration of any action will be determined based on the nature and impact of the violation.
+
+## Attribution
+
+This Code of Conduct is adapted from the Contributor Covenant, version 2.1.  
+https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)